lhotse/recipes/iwslt2022_dialect.py

```diff
@@ -0,0 +1,469 @@
+"""
+LDC2022E01 (Tunisian Arabic Dialect) is a speech translation corpus
+LDC2022E02 (Tunisian Arabic Dialect Eval Set)
+"""
+import logging
+import re
+from collections import defaultdict
+from typing import Dict, Iterable, List, Optional, Union
+from pathlib import Path
+from concurrent.futures.thread import ThreadPoolExecutor
+import tqdm
+import soundfile as sf
+import string
+
+try:
+    import pyarabic.number as number
+    from pyarabic import araby
+except ImportError:
+    import pip
+
+    pip._internal.main(["install", "pyarabic"])
```

Comment:
Smart trick, but as a user I would be extremely surprised if importing Lhotse triggered a pip install command. Please handle missing optional dependencies in the same way as throughout the rest of the code, see e.g. here:

https://github.com/lhotse-speech/lhotse/blob/933fb64834d5bc603db1ef8ffd1d2004ae75497a/lhotse/recipes/gigaspeech.py#L38-L43